pyproject.toml

```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
 name = "monarch-ingest"
-version = "0.1.0"
+version = "0.1.1"
 description = ""
-authors = ["Kevin Schaper <kevinschaper@gmail.com>"]
+authors = ["Kevin Schaper <kevinschaper@gmail.com>", "The Monarch Initiative <info@monarchinitiative.org>"]
```

Comment:
haha, feel free to make it it just info@monarchinitiative.org

Reply:
I added all of us!